idaes/power_generation/properties/flue_gas_ideal.py

```diff
@@ -419,11 +560,10 @@ def initialize(
             else:
                 res = "skipped"
             init_log.info_high(
-                "Initialization Step 2 {}.".format(idaeslog.condition(res)))
+                "Initialization Step 2 {}.".format(idaeslog.condition(res))
+            )
 
-        init_log.info(
-            'Initialisation Complete, {}.'.format(
-                idaeslog.condition(res)))
+        init_log.info("Initialisation Complete, {}.".format(idaeslog.condition(res)))
```

Comment:
We should also add an `InitializationError` here if the final solve was not optimal:

```
if not check_optimal_termination(res):
    raise InitializationError()
```